classes/Entities/Course.php

```diff
@@ -31,10 +31,18 @@ public function category()
     }
 
     /**
-     * @return mixed
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
      */
     public function courseCohortSyncs()
     {
         return $this->hasMany(CourseCohortSync::class,'courseid','id');
     }
+
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function enrol(){
```

Comment:
it could be "enrolments" as it has a hasMany relationship, as it stating multiple values